### DISCUSSION TITLE
Interactive Indicators

### DISCUSSION THREAD

**Discussion by MisterOzzy** [reactivemarkets/react-financial-charts#471] [General]
In description you have 'Interactive Indicators' like Trend line, Fibo retracement and so on? where can i find examples how to use them? or please just write a few examples how to use them

**Comment by markmcdowell** [comment]
The code is in the interactive project https://github.com/reactivemarkets/react-financial-charts/tree/master/packages/interactive/src

I've yet to add the examples, hopefully this week.

**Reply by MisterOzzy** [reply]
many thanks, it will be very very useful for me as I started using your library

**Reply by oneart-dev** [reply]
At least one example with Interactive Indicators would help a lot

**Reply by paulbse** [reply]
@MisterOzzy @oneart-dev @markmcdowell Did you find any examples ? :)
